app/controllers/peer_reviews_controller.rb

```diff
@@ -89,6 +90,13 @@ def unassign(reviewers_to_remove_from_reviewees_map)
         pr.destroy
```

Comment:
I don't know that it is easy to get a list of all the PeerReviews to remove here, so I wouldn't worry about it.